<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>500</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="maximumSize">
    <size>
     <width>700</width>
     <height>500</height>
    </size>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>271</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <widget class="QWidget" name="searchWidget" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>5</y>
          <width>251</width>
          <height>30</height>
         </rect>
        </property>
        <widget class="QPushButton" name="searchPButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>5</x>
           <y>5</y>
           <width>20</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLineEdit" name="searchLEdit">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>5</y>
           <width>211</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="qqTWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30000</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="qqFriendTab">
        <attribute name="title">
         <string/>
        </attribute>
        <widget class="QQListWidget" name="friendLWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>271</width>
           <height>361</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="qqGroupTab">
        <attribute name="title">
         <string/>
        </attribute>
        <widget class="QQListWidget" name="groupLWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>271</width>
           <height>361</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="qqClosestTab">
        <attribute name="title">
         <string/>
        </attribute>
        <widget class="QQListWidget" name="closestLWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>271</width>
           <height>361</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="stateWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <widget class="QPushButton" name="addFriendPButton">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>10</y>
          <width>25</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="statusButton">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>15</y>
          <width>12</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="iconSelfPButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>5</y>
          <width>30</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="optionPButton">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>10</y>
          <width>25</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QQListWidget</class>
   <extends>QListWidget</extends>
   <header location="global">qqlistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
